egui-wgpu/src/renderer.rs

```diff
@@ -128,7 +128,7 @@ pub struct RenderPass {
     /// Map of egui texture IDs to textures and their associated bindgroups (texture view +
     /// sampler). The texture may be None if the TextureId is just a handle to a user-provided
     /// sampler.
-    textures: HashMap<egui::TextureId, (Option<wgpu::Texture>, wgpu::BindGroup)>,
+    pub textures: HashMap<egui::TextureId, (Option<wgpu::Texture>, wgpu::BindGroup)>,
```

Comment:
should everyone be able to just mutate this as is? Would it be better to use accessor functions instead?

Reply:
OK, I made `textures` private again, and I also made all of the functions that users shouldn't need to call private to the crate.

Reply:
great!